loc/tests/test_schema.py

```diff
@@ -332,3 +334,69 @@ def test_email_letter_works(db, graphql_client, mailoutbox):
     )['data']['emailLetter']
     assert result == {'errors': [], 'recipients': ['boop@jones.com']}
     assert len(mailoutbox) == 1
+
+
+RA_DATA_QUERY = '''
+query {
+    session {
+        onboardingInfo {
+            hasCalled311
+        }
+    }
+}
+'''
+
+RA_MUTATION = """
+mutation {
+    reliefAttempts(input: {}) {
+        errors { field, messages }
+        session {
+            onboardingInfo {
+                hasCalled311
+            }
+        }
+    }
+}
+"""
+
+
+def _get_relief_attempts_info(graphql_client):
+    return
+    graphql_client.execute(RA_DATA_QUERY)['data']['session']['onboardingInfo']['hasCalled311']
+
+
+def _exec_relief_attempts_form(graphql_client, **input_kwargs):
+    return graphql_client.execute(
+        RA_MUTATION,
+        variables={'input': {
+            **input_kwargs
+        }}
+    )['data'][f'output']
+
+
+def test_relief_attempts_form_validates_data(db, graphql_client):
+    oi = OnboardingInfoFactory()
+    graphql_client.request.user = oi.user
+    ob = _exec_relief_attempts_form(graphql_client, hasCalled311='Boop')
+    assert len(ob['errors']) > 0
+    assert _get_relief_attempts_info(graphql_client) is None
+
+
+def test_relief_attempts_form_saves_data_to_session(db, graphql_client):
+    oi = OnboardingInfoFactory()
+    graphql_client.request.user = oi.user
+    ob = _exec_relief_attempts_form(graphql_client, hasCalled311='True')
+    print(ob)
+    assert ob['errors'] == []
+    assert ob['session']['onboardingInfo']['hasCalled311'] is True
```

Comment:
@toolness I'm getting some errors here in that when I execute the form mutation in these tests, it doesn't actually recognize my setting of the `hasCalled311` variable. I implemented `_exec_relief_attempts_form` and the other tests based on the ones I wrote for Rent History so having a hard time figuring out what the missing piece is here. If you have any suggestions, let me know!